documentation/docs/concepts/decentralized_identifiers/update.mdx

```diff
@@ -46,363 +50,244 @@ methods allow them to be used by more than one verification relationship.
 Upon creating a verification method using the identity framework, specifying the `MethodScope` option will result in an embedded verification method. Leaving that option unset will create the verification method as
 a map entry of the `verificationMethod` property. Verification relationships can be added afterwards using references.
 
-:::warning
+:::note
 
-Any update to the DID document must be signed using a verification method with `capability invocation` relationship to be valid. Removing all capability invocation verification methods
-disallows any further updates to the document.
+Updates to the DID Document are done through a state transition of the [Alias Output](../../specs/did/iota_did_method_spec#alias-output). The State Controller does not have to be listed as a Capability Invocation verification method.
```

Comment:
The capability invocation reference is only relevant to previous users, it's more accurate to say it doesn't have to be included at all, if we keep that sentence.

```suggestion
Updates to the DID Document are done through a state transition of the [Alias Output](../../specs/did/iota_did_method_spec#alias-output) by its state controller. The public key or address of the state controller does not need to be a verification method in the DID Document, since it is defined in the containing Alias Output.
```